drops.js

```diff
@@ -0,0 +1,27 @@
+class Dropdown {
+    constructor(element) {
+        this.element = element;
+        console.log('this', this);
+        this.child = document.querySelector('.Dropdown__Content');
+        this.element.addEventListener('click', () => {
+            // let children = this.element.children;
+            // children[1].classList.add('Dropdown--Display'); // for use with getElementsByClassName array
+            this.child.classList.toggle('Dropdown--Display');
+        });
+    }
+}
+
+let dropdown = document.getElementsByClassName('Dropdown');
+dropdown = Array.from(dropdown).map(dd => {
+    return new Dropdown(dd);
+});
+
+let secLower = document.querySelector('.Section--Lower');
+secLower.addEventListener('click', () => {
+    let ddContent = document.querySelector('.Dropdown--Display');
+    if (ddContent) {
+        console.log(true);
+        ddContent.classList.remove('Dropdown--Display');
+        console.log(ddContent.classList);
+    }
+});
```

Comment:
Okay, I think it would be a good idea for you and I (or Dan, if you prefer) to hop on a zoom chat about this. You're very close, but I have a feeling you might be doing something that I always do --> make things more complex than they need to be.  I think that's what Ivan was trying to express to you last night as well. Spend some time on this, and I would just to focus on trying to simplify things. You got this!

Reply:
Thanks Greg, I just did another commit about 30 minutes ago.  Not sure if you saw those changes...